pkg/cmd/server/start/start_master.go

```diff
@@ -613,6 +613,8 @@ func startControllers(oc *origin.MasterConfig, kc *kubernetes.MasterConfig) erro
 		kc.RunPersistentVolumeClaimRecycler(oc.ImageFor("recycler"), recyclerClient, oc.Options.PolicyConfig.OpenShiftInfrastructureNamespace)
 		kc.RunGCController(gcClient)
 
+		kc.RunServiceLoadBalancerController()
```

Comment:
> Yeah, there's still a mix of the old and new implementations. Been trying to catch anything new going in.

Agree.

As a note for whoever does the rebase, we need to sort out what we do about shared cache ACL-ing.  @liggitt